Sources/Utility/Process.swift

```diff
@@ -95,6 +105,9 @@ public final class Process: ObjectIdentifierProtocol {
 
     /// The output read from the process, if redirected.
     private var output: [Int8] = []
+    
+    /// Queue to protect concurrent reads.
+    private let serialQueue = DispatchQueue(label: "org.swift.swiftpm.process")
```

Comment:
I don't think we should try and make the labels include the process ID, they are always scoped to a single process.